lib/ansible/modules/cron.py

```diff
@@ -73,34 +73,34 @@
     default: no
   minute:
     description:
-      - Minute when the job should run ( 0-59, *, */2, etc )
+      - Minute when the job should run ( C(0-59), C(*), C(*/2), and so on ).
```

Comment:
```suggestion
      - Minute when the job should run (C(0-59), C(*), C(*/2), and so on).
```
Same for the other entries.

Reply:
all ingenious is simple:)